host-bmc/host_pdr_handler.cpp

```diff
@@ -806,13 +806,27 @@ void HostPDRHandler::processHostPDRs(mctp_eid_t /*eid*/,
                                     }
                                 }
                             }
+                            modifiedCounter--;
                         }
                     }
-                    else
+                    else if ((isHostPdrModified != true) &&
```

Comment:
Can you add a comment here why this change was made and if there is any optimal solution other than this